src/Components/Home.js

```diff
@@ -79,20 +85,20 @@ export default function Example() {
 
 
             <div className="mx-auto max-w-2xl py-32 sm:py-48 lg:py-0">
-        <div className="text-center">
-          <h1 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-4xl">
-            One platform to streamline all workflows
-          </h1>
-          <p className="mt-2 text-lg leading-8 text-gray-600">
-            Your all-in-one platform to manage projects, organize work, enhance collaboration, and accelerate execution across all departments.
-          </p>
-        </div>
-        {showLogin ? (
-        <LoginForm toggleSignup={toggleForm} />
-      ) : (
-        <SignupForm toggleLogin={toggleForm} />
-      )}
-      </div>
+              <div className="text-center">
+                 <h1 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-4xl">
+                     One platform to streamline all workflows
+                  </h1>
+                  <p className="mt-2 text-lg leading-8 text-gray-600">
+                  Your all-in-one platform to manage projects, organize work, enhance collaboration, and accelerate execution across all departments.
```

Comment:
Please Format the code in proper standard format and remove black spaces and comments

Reply:
Done